README.md

```diff
@@ -6,12 +6,25 @@
        src="https://scan.coverity.com/projects/12056/badge.svg"/>
 </a>
 
-## Features
-Camera streaming daemon currently supports the following features:
+The *Camera Streaming Daemon* is an extensible Linux camera server for interfacing cameras with the Dronecode platform. 
 
- - Video Streaming: RTSP
- - Mavlink Camera Protocol: https://mavlink.io/en/protocol/camera.html
- - Image Capture
+It provides a [MAVLink Camera Protocol](https://mavlink.io/en/protocol/camera.html) compatible API for video and image capture etc., and an RTSP service for advertising and sharing video streams. The server can connect to and manage multiple cameras, and has been designed so that it can be extended to support new camera types and protocols when needed. 
+
+> **Tip** The daemon is the easiest way for Camera OEMs to interface with the Dronecode platform. Many cameras will just work "out of the box". At most OEMs will need to implement a camera integration layer.
+
+## Overview
+
+The diagram below shows the main components of a system running the *Camera Streaming Daemon* on a Linux companion computer.
+
+<img src="https://github.com/hamishwillee/camera-streaming-daemon/blob/readme_overview/docs/assets/camera_streaming_daemon_overview.png" width="600px" title="Camera streaming daemon overview" />
```

Comment:
I know this is the WIP URL. Just a friendly reminder to make it point to the asset in the main repo ;)

Reply:
Thanks, will fix in a second. I had actually missed this :-(